src/lib/tuya.ts

```diff
@@ -1098,6 +1098,15 @@ const tuyaTz = {
             await entity.read('genLevelCtrl', [0xfc00]);
         },
     } satisfies Tz.Converter,
+    min_brightness2: {
```

Comment:
```suggestion
    min_brightness_2: {
```

And could you rename the existing `min_brightness` to `min_brightness_1`